packages/specification/schemas/components/MedicationRequest/DispenseNotificationMedicationRequest.yaml

```diff
@@ -0,0 +1,400 @@
+type: object
+required:
+  - resourceType
+  - identifier
+  - status
+  - intent
+  - category
+  - medicationCodeableConcept
+  - subject
+  - authoredOn
+  - requester
+  - groupIdentifier
+  - courseOfTherapyType
+  - dosageInstruction
+  - dispenseRequest
+  - substitution
+description: |
+  A MedicationRequest object is required.
+  An entity representing the line items on a prescription in the context of a dispense notification.
+  An EPS prescription must contain at least one and up to four MedicationRequest objects.
+properties:
+  resourceType:
+    type: string
+    description: FHIR resource type
+    enum: [MedicationRequest]
+  id:
+    type: string
+    example: "m3"
+  extension:
+    type: array
+    description: A FHIR extension array.
+    items:
+      oneOf:
+      - $ref: ../_extensions/ControlledDrug.yaml
+      - $ref: ../_extensions/DM-ResponsiblePractitioner.yaml
+      - $ref: ../_extensions/DMPrescriptionStatusHistory.yaml
```

Comment:
As I've noted below, we don't need to reference this within the MedicationRequest in the context of dispensing interactions, given that this pertains to a cancel response - which won't be visible to dispensers when they're dispensing prescriptions.

Reply:
Removed reference to DMPrescriptionStatusHistory extension.

Reply:
Removed reference to DMPrescriptionStatusHistory extension.